Because not all Void kernels may support zstd, change the default initramfs
compression to gzip.

diff -ur a/man/mkinitcpio.conf.5.txt b/man/mkinitcpio.conf.5.txt
--- a/man/mkinitcpio.conf.5.txt	2021-02-16 21:37:31.000000000 -0500
+++ b/man/mkinitcpio.conf.5.txt	2021-05-17 09:33:32.418504652 -0400
@@ -55,7 +55,7 @@
 	Defines a program to filter the generated image through. The kernel
 	understands the compression formats yielded by the *zstd*, *gzip*, *bzip2*,
 	*lz4*, *lzop*, *lzma*, and *xz* compressors. If unspecified, this setting
-	defaults to *zstd* compression. In order to create an uncompressed image,
+	defaults to *gzip* compression. In order to create an uncompressed image,
 	define this variable as *cat*.
 +
 It's not hard to realize that a filter such as a *tac* or *rev* will cause
diff -ur a/mkinitcpio b/mkinitcpio
--- a/mkinitcpio	2021-02-16 21:37:31.000000000 -0500
+++ b/mkinitcpio	2021-05-17 09:34:15.970588222 -0400
@@ -508,7 +508,7 @@
         die 'Unable to write to %s' "$_optgenimg"
     fi
 
-    _optcompress=${_optcompress:-${COMPRESSION:-zstd}}
+    _optcompress=${_optcompress:-${COMPRESSION:-gzip}}
     if ! type -P "$_optcompress" >/dev/null; then
         warning "Unable to locate compression method: %s" "$_optcompress"
         _optcompress=cat
diff -ur a/mkinitcpio.conf b/mkinitcpio.conf
--- a/mkinitcpio.conf	2021-02-16 21:37:31.000000000 -0500
+++ b/mkinitcpio.conf	2021-05-17 09:34:24.752605714 -0400
@@ -52,7 +52,7 @@
 HOOKS=(base udev autodetect modconf block filesystems keyboard fsck)
 
 # COMPRESSION
-# Use this to compress the initramfs image. By default, zstd compression
+# Use this to compress the initramfs image. By default, gzip compression
 # is used. Use 'cat' to create an uncompressed image.
 #COMPRESSION="zstd"
 #COMPRESSION="gzip"
